### DISCUSSION TITLE
Fully supported parts?

### DISCUSSION THREAD

**Discussion by maxdepoli** [OrcaSlicer/OrcaSlicer#1871] [Q&A] [answered]
Hi all, is there a way to prevent automatic drop of all models on the build plate? 
I wish to print a part fully on supports, for all the faces are slightly convex, and it is annoying to remove very short supports.

**Comment by Glenn1963** [comment, marked as answer]
If you put one object, even just Add a Primitive, on the bed first, you can then right-click that and load anything else as a "Part". A Part can be moved anywhere. You may need to switch to manipulating Objects and get just the part in question selected, but it can be done.
![image](https://github.com/SoftFever/OrcaSlicer/assets/17497231/035e65fd-1686-4e94-9311-ebc1f192c223)
